setup.py

```diff
@@ -85,7 +85,8 @@ def main():
                 "urwid==1.3.1"
             ],
             "wx": [
-                "darkdetect==0.3.1"
+                "darkdetect==0.3.1",
```

Comment:
Why do you require a specific version? Why not always picking the latest one? The API looks simple and so is not going to change, no?